modules/jvm/module.yaml

```diff
@@ -16,10 +16,12 @@ envs:
 - name:         JBOSS_CONTAINER_JAVA_JVM_MODULE
   value:        /opt/jboss/container/java/jvm
 - name:         JAVA_OPTS
-  description:  JVM options passed to the `java` command.
+  description:  If set, this prevents all default and generated options (such
+    as JVM and GC tuning, proxy settings, etc.) from being applied. Instead the
+    user should specify exactly the `java` options they want in this variable.
   example:      "-verbose:class"
 - name:         JAVA_OPTS_APPEND
-  description:  User specified Java options to be appended to generated options in JAVA_OPTS.
+  description:  User specified Java options to be appended to the generated options.
```

Comment:
Please add that this variable has no effect if `JAVA_OPTS` is set in the description.

Reply:
I don't mind doing so, and I will. I originally decided not to because I wasn't also going to annotate all the other variables that are disabled by specifying `JAVA_OPTS`, but, due to their similar name and behaviour I can see that this one should be treated differently.